docs/development_suite/api-console/api-design/merge_collaboration.md

```diff
@@ -91,6 +91,14 @@ Entering one of them, to resolve a diff, you can:
 
 In every section, navigate from one change to the following/previous one thanks to the related navigation buttons. If there are no changes in the current section, the navigation buttons will be disabled.
 
+In case you want to accept all incoming or current changes inside a specific section of the merge modal, to speed-up your merge process you can leverage on two different buttons:
```

Comment:
```suggestion
In case you want to accept all incoming or current changes inside a specific section of the merge modal, to speed up your merge process you can leverage two different buttons:
```